index.html

```diff
@@ -196,5 +70,8 @@
               <div class="fb-like" data-href="https://www.facebook.com/metakgp/" data-layout="button_count" data-action="like" data-size="small" data-show-faces="false" data-share="false" data-colorscheme="dark"></div>
              </footer>
         </div>
-    </body>
+      </div>
+    </div>
+    <script src="local-storage.js"></script>
```

Comment:
There's no local-storage.js file being used. I'll fix this along with UI part. Sending a PR in an hour.

Reply:
Great, rebase onto the latest master. Thanks! :+1: